Gemfile

```diff
@@ -100,21 +100,35 @@ end
 # BULLET TRAIN GEMS
 # This section is the list of Ruby gems included by default for Bullet Train.
 
+# We use a constant here so that we can ensure that all of the bullet_train-*
+# packages are on the same version.
+BULLET_TRAIN_VERSION = "1.3.24"
```

Comment:
Love this 💯
Will be easy for developers to spot and change too.

Reply:
This is how I do it.